openedx.yaml

```diff
@@ -0,0 +1,9 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: porta
+oeps:
+  oep-30:
+    state: True
+openedx-release: {ref: master}
```

Comment:
This repo should be part of Open edX?

Reply:
Yeah, I believe it will be. Is there something that should be changed here? This was just built from the cookie cutter.

Reply:
This is going to be how people configure the leaner-portal for reference.

Reply:
If it should be included, then we are all good. I check in on new openedx.yaml files, since people sometimes don't understand the implications.

Reply:
Can you add a mention of this new thing on the Juniper wiki page?